apps/bblcricket/bbl_cricket.star

```diff
@@ -0,0 +1,771 @@
+load("render.star", "render")
+load("http.star", "http")
+load("humanize.star", "humanize")
+load("time.star", "time")
+load("encoding/json.star", "json")
+load("schema.star", "schema")
+load("math.star", "math")
+
+"""
+Applet: BBL Cricket
+Summary: Displays scores for the selected BBL team
+Description: This app takes the selected team and displays the current match situation - showing overall team score, batsmen scores, lead/deficit, overs bowled, run rate and required run rate for the team batting second. If a match for the selected team has just completed, it will show the match result or if there is an upcoming match it will show the teams win-loss record and the scheduled start time of the match, in the users timezone. If there is nothing coming up in the next day or so (as determined by the Cricinfo API), it will no show that there are no matches scheduled.
+Author: M0ntyP 
+"""
+
+LiveGames_URL = "https://hs-consumer-api.espncricinfo.com/v1/pages/matches/current?lang=en&latest=true"
+Standings_URL = "https://hs-consumer-api.espncricinfo.com/v1/pages/series/standings?lang=en&seriesId=1324623"
+DEFAULT_TEAM = "4848"
+
+DEFAULT_LOCATION = """ 
+{
+    "description": "Adelaide, SA, Australia",
+    "locality": "Adelaide",
+    "timezone": "Australia/Adelaide"
+}
+"""
+
+def main(config):
+    SelectedTeam = config.get("TeamList", DEFAULT_TEAM)
+    SelectedTeam = int(SelectedTeam)
+
+    Standings_JSON = http.get(Standings_URL).json()
```

Comment:
We'll need some caching around these calls to make sure we're not hitting the API to frequently.

Reply:
Can do, I'll fix that up